src/parameterized_tendencies/les_sgs_models/smagorinsky_lilly.jl

```diff
@@ -166,7 +166,7 @@ function vertical_smagorinsky_lilly_tendency!(Yₜ, Y, p, t, ::SmagorinskyLilly)
         ᶜ∇ᵥρD∇χₜ = @. ᶜtemp_scalar = ᶜdivᵥ_ρχ(-(ᶠρ * ᶠD_smag * ᶠgradᵥ(ᶜχ)))
         @. ᶜρχₜ -= ᶜ∇ᵥρD∇χₜ
         # Rain and snow does not affect the mass
-        if χ_name ∉ (:q_rai, :q_sno)
+        if χ_name ∉ (:q_liq, :q_ice, :q_rai, :q_sno)
```

Comment:
I was wondering if here and in other files we could write it as `χ_name == q_tot` . I'm thinking ahead about the other microphysics or other tracers that we will will be adding. Maybe its safer to make it an explicit "opt-in" rather than "opt-out". 

I think it's likely we will always have a `q_tot`. But not certain what other cloud tracers will be there

Reply:
If we ever have other tracers e.g. aerosols they should be included in the mass tendency as well. But I don't think we will have them anytime soon, so up to you.

Reply:
opt-in notation is concise, but I found the opt-out version is clearer when checking tendencies at a glance, especially in the implicit solver; cc @trontrytel updated now to use `opt-in` logic.